lmdb-sys/src/ffi.rs

```diff
@@ -62,6 +62,9 @@ extern "C" {
     pub fn mdb_version(major: *mut ::libc::c_int, minor: *mut ::libc::c_int, patch: *mut ::libc::c_int) -> *mut ::libc::c_char;
     pub fn mdb_strerror(err: ::libc::c_int) -> *mut ::libc::c_char;
     pub fn mdb_env_create(env: *mut *mut MDB_env) -> ::libc::c_int;
+    #[cfg(windows)]
+    pub fn mdb_env_open(env: *mut MDB_env, path: *const ::libc::c_char, flags: ::libc::c_uint, mode: ::libc::c_int) -> ::libc::c_int;
```

Comment:
I think it would reduce the number of `cfg`s required to declare a platform specific `lmdb_sys::mode_t` typedef (as lmdb itself does) and use that instead.  Something like:

```
#[cfg(unix)]
pub type mode_t = ::libc::mode_t;
#[cfg(windows)]
pub type mode_t = ::libc::c_int;
```